libs/utils/src/id.rs

```diff
@@ -168,51 +168,51 @@ macro_rules! zid_newtype {
 /// branches? If they pick the same one, and later try to push the
 /// branches to the same remote server, they will get mixed up.
 ///
-/// To avoid those issues, Zenith has its own concept of timelines that
+/// To avoid those issues, Neon has its own concept of timelines that
 /// is separate from PostgreSQL timelines, and doesn't have those
-/// limitations. A zenith timeline is identified by a 128-bit ID, which
+/// limitations. A Neon timeline is identified by a 128-bit ID, which
 /// is usually printed out as a hex string.
 ///
 /// NOTE: It (de)serializes as an array of hex bytes, so the string representation would look
 /// like `[173,80,132,115,129,226,72,254,170,201,135,108,199,26,228,24]`.
-/// See [`ZId`] for alternative ways to serialize it.
+/// See [`Id`] for alternative ways to serialize it.
 #[derive(Clone, Copy, PartialEq, Eq, Hash, Ord, PartialOrd, Serialize, Deserialize)]
-pub struct ZTimelineId(ZId);
+pub struct TimelineId(Id);
 
-zid_newtype!(ZTimelineId);
+id_newtype!(TimelineId);
 
-/// Zenith Tenant Id represents identifiar of a particular tenant.
+/// Neon Tenant Id represents identifiar of a particular tenant.
 /// Is used for distinguishing requests and data belonging to different users.
 ///
 /// NOTE: It (de)serializes as an array of hex bytes, so the string representation would look
 /// like `[173,80,132,115,129,226,72,254,170,201,135,108,199,26,228,24]`.
-/// See [`ZId`] for alternative ways to serialize it.
+/// See [`Id`] for alternative ways to serialize it.
 #[derive(Clone, Copy, PartialEq, Eq, Hash, Serialize, Deserialize, PartialOrd, Ord)]
-pub struct ZTenantId(ZId);
+pub struct TenantId(Id);
 
-zid_newtype!(ZTenantId);
+id_newtype!(TenantId);
 
-// A pair uniquely identifying Zenith instance.
+// A pair uniquely identifying Neon instance.
 #[derive(Debug, Clone, Copy, PartialOrd, Ord, PartialEq, Eq, Hash, Serialize, Deserialize)]
 pub struct ZTenantTimelineId {
```

Comment:
Needs rename too

Reply:
Thank you, almost forgot it, thinking of search regex instead.